test/100-basic-name-resolution.bats

```diff
@@ -49,10 +53,14 @@ load helpers
 	a1_pid=$CONTAINER_NS_PID
 	run_in_container_netns "$a1_pid" "dig" "+short" "aone" "@$gw"
 	assert "$ip_a1"
+	# Set recursion bit
+        assert "$output" !~ "WARNING: recursion requested but not available"
```

Comment:
Nit: spacing on these asserts is weird - doesn't match the rest of the file

Reply:
My bad, i used spaces instead of tabs and added extra space.